openebs/openebs-provisioner.go

```diff
@@ -164,6 +166,16 @@ func (p *openEBSProvisioner) Provision(options controller.VolumeOptions) (*v1.Pe
 		volAnnotations["alpha.dashboard.kubernetes.io/links"] = "{" + strings.Join(userLinks, ",") + "}"
 	}
 
+	fsType := ""
```

Comment:
How about instead of traversing map we do:
```
if fsType=options.Parameter["openebs.io/fstype"]; fsType == ""{
    fsType = defaultFSType
}
```

Reply:
We can validate fsType if we use a const map ^^^

Reply:
Yes , if we have only 1 key to be be parsed , traversing of map is not required.

Reply:
We will have some more key/value pairs related to volume policy as a different cases which will be required to add here.